Resolve https://github.com/ruflin/Elastica/issues/1737: handle 413 response (#2055)
ruflin/Elastica: CHANGELOG.md

2011-03-21
- ChildrenAggregation added - https://www.elastic.co/guide/en/elasticsearch/guide/current/children-agg.html

> 2022-03-21

Reviewer: Can you put this on the top of the changelog under Unreleased in the right category?

> - Added `\Elastica\Exception\RequestEntityTooLargeException`
> - Adjusted `\Elastica\Bulk`, throw RequestEntityTooLargeException if response code 413

Reviewer 2: ```suggestion
- Add throwing `\Elastica\Exception\RequestEntityTooLargeException` on HTTP-413 responses in `\Elastica\Bulk`
```
